### PR TITLE
Fix BC break for factory setter

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

File: src/DependencyInjection/HautelookAliceExtension.php [hunk 1]
```diff
@@ -59,15 +61,44 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
 
+        // Deprecated factory methods handling.
+        // To be removed and set directly on config file when bumping Symfony requirements to >=2.6
+        $aliceFakerDefinition = $container->getDefinition('hautelook_alice.faker');
+        if (method_exists($aliceFakerDefinition, 'setFactory')) {
+            $aliceFakerDefinition->setFactory(array('Faker\Factory', 'create'));
```
Comment (theofidry) [inline src/DependencyInjection/HautelookAliceExtension.php]: let's use with the short array notation, the minimum requirement is PHP 5.4 anyway although Symfony still officially supports 5.3

**Comment by Soullivaneuh** [comment]
@theofidry done.

**Comment by theofidry** [comment]
You might want to rebase now to get the full Travis build

**Comment by Soullivaneuh** [comment]
Done.

**Comment by Soullivaneuh** [comment]
Tests passes. :+1:

**Pull request merged**